changes/400.added

```diff
@@ -0,0 +1 @@
+Add ability to customize `NautobotAdapter` loading on a per-model basis.
```

Comment:
isn't this file 566?

Reply:
Fragment files use issue number:
https://docs.nautobot.com/projects/core/en/stable/development/core/?h=changelog#creating-changelog-fragments